### PR TITLE
PHPDocs fixes and strict type added  [Widget module]

### PR DESCRIPTION
**Pull request by ghost**
### Description
- Small PHPDocs fixes in Widget module
- Added strict types

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @SikailoISM. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/code/Magento/Widget/Model/ResourceModel/Widget/Instance.php [hunk 1]
```diff
@@ -29,10 +31,10 @@ protected function _construct()
     /**
      * Perform actions after object load
      *
-     * @param \Magento\Widget\Model\Widget\Instance $object
```
Comment (sivaschenko) [inline app/code/Magento/Widget/Model/ResourceModel/Widget/Instance.php]: I'd prefer having as precise as possible annotation, is this method used for processing any abstract models except `Magento\Widget\Model\Widget\Instance` ?

Comment (ghost) [reply app/code/Magento/Widget/Model/ResourceModel/Widget/Instance.php]: fixed

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-4159](https://jira.corp.magento.com/browse/ENGCOM-4159) has been created to process this Pull Request

**Comment by sivaschenko** [comment]
Hi @SikailoISM can you please remove strict types for method parameters, as these changes are backward incompatible.

**Pull request closed without merging**